hello Microverse 2: linterns, html, css, gitignore

Wikitelmex opened the pull request:
## Modifications
-Added lintern.yml file to checkouts
-index.html file added with a header
-style.css file added and added some header styles
.gitignore updated to avoid module folder

wikitelmex1 committed:
changes on branch to be revied check for index.html code, header added

algerina commented on readme.md:
```diff
@@ -0,0 +1,66 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
```
Hi

Wikitelmex commented on htmlcss-lintern.json:
```diff
@@ -0,0 +1 @@
+some lintern code
```
add real code to lintern

ahmetbozaci commented on readme.md:
```diff
@@ -0,0 +1,66 @@
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/wikitelmex)
+
+
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
- [ ]  You should update your README file. You can use this [template](https://github.com/microverseinc/readme-template) and make sure there are no  broken links(example screenshot, licence) and empty sections. If you won't use some sections, delete them. 
- [ ] I recommend using as many readme sections as you can.
- [ ] You should change the readme template's sentence with your sentence.

Wikitelmex replied:
> # STATUS: Changes Required ♻️ -first
> ### Hello @Wikitelmex ✋🏼 Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.
> ### 💯 Nice to have 💯
> * Your git flow is correct ✔️
> * You create html and css files according to the requirement.✔️
> 
> ### ♻️ Request changes ♻️
> * [ ]  Add a short summary of changes introduced in the pull request. Please, check this [lesson in curriculum](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)
> * [ ]  You didn't set up linter files. Please check this [repository](https://github.com/microverseinc/linters-config/tree/master/html-css) and set up linter files correctly.  Make sure all linter tests pass.
>   ![Screenshot from 2021-08-10 20-01-09](https://user-images.githubusercontent.com/38635158/128920254-f82a37e1-86e9-4562-9b0c-ee0a934bfc6e.png)
> * [ ]  _**Check the comments under the review.**_
> 
> ### OPTIONAL SUGGESTIONS
> #### These are optional but highly recommended.
> * [ ]  Your commit messages should always start  with capital letters and you should write an imperative tense. I see some of them are not like that. You can change them by using `git rebase`.
> * [Video - How to change commit messages](https://www.loom.com/share/00771f369b7b47fc968c019ecf0d3ba6)
> * [Article - How to create good commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/)
> 
> ### Cheers and Happy coding! 💻 👏
> ![almost](https://camo.githubusercontent.com/1ebf31158d6fdf5bc2f2a45d75816ebcc5079f54ecba802890dc4f85998f73ef/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6c30486c443563695757686c7241774f512f67697068792e676966)
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_
> 
> _If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [ahmetbozaci](https://github.com/ahmetbozaci), I will receive a notification and will respond._
> 
> [GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)
> 
> _Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

I have some issues running the linterns here on Github, looks like the server don't response, but, the lintern file added as the tutorial says.

Wikitelmex replied:
I have some issues running the linterns here on Github, looks like the server doesn't respond, but, the lintern file was added as the tutorial says.

Updated the README file

ahmetbozaci commented on index.html:
```diff
@@ -0,0 +1 @@
+<h1>Hello Microverse</h1>
```
- [ ] You should  add some style to this tag. [Requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#:~:text=A%20.css%20file%20that%20includes%20one%20class%20with%20styles%20for%20your%20header%20(make%20it%20in%20your%20favorite%20color).)

Wikitelmex replied:
style added in style.css as a class and implemented inside the index.html h1 tag

ahmetbozaci reviewed (changes requested):
# STATUS: Changes Required  ♻️

### Hello @Wikitelmex ✋🏼  Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

### 💯  Nice to have 💯
 
- Your git flow is correct ✔️
- You create html and css files according to the requirement.✔️

### ♻️ Request changes ♻️
- [ ] Add a short summary of changes introduced in the pull request. Please, check this [lesson in curriculum](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)
- [ ] You didn't set up linter files. Please check this [repository](https://github.com/microverseinc/linters-config/tree/master/html-css) and set up linter files correctly.  Make sure all linter tests pass.
![Screenshot from 2021-08-10 20-01-09](https://user-images.githubusercontent.com/38635158/128920254-f82a37e1-86e9-4562-9b0c-ee0a934bfc6e.png)



- [ ] _**Check the comments under the review.**_


### OPTIONAL SUGGESTIONS

#### These are optional but highly recommended.

- [ ] Your commit messages should always start  with capital letters and you should write an imperative tense. I see some of them are not like that. You can change them by using `git rebase`. 
- [Video - How to change commit messages](https://www.loom.com/share/00771f369b7b47fc968c019ecf0d3ba6)
- [Article - How to create good commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/)


### Cheers and Happy coding! 💻 👏
![almost](https://media.giphy.com/media/l0HlD5ciWWhlrAwOQ/giphy.gif)

___
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [ahmetbozaci](https://github.com/ahmetbozaci), I will receive a notification and will respond._

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

wikitelmex1 committed:
Linter modifications

-installed lighthouse
-check for errors wit lint
-errors fixed

wikitelmex1 committed:
Create linterns.yml

lintern.yml added inside .github/workflows

wikitelmex1 committed:
Update .gitignore

wikitelmex1 committed:
css class for h1

added an style class called main-header, to improve the h1 label in html with a blue color

barackm reviewed (approved):
Hi @Wikitelmex , 👋 

# STATUS: Approved  🎉 
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 

![](https://media.giphy.com/media/vmtxnxveVUodG/giphy.gif)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] Please it's good to remove the node_modules folder from the repository and add it to the .gitignore file because it's a very big folder and it can get regenerated by running `npm install `. this makes the review so difficult as well.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Wikitelmex commented:
@barackm thank you very much, I already merged, I deleted module folder from github repo, is anything else that I must to do? because I still have the yellow bag in my course. Thanks again

![image](https://user-images.githubusercontent.com/59240486/128947690-1031c726-3af6-4a57-bb35-6f00f2f8a3a9.png)

barackm commented:
Hi @Wikitelmex, now as the badge says `submit` you can submit the project, if you click on actions you will find an option  `submit` click on it and then fill out the form with the information they will request.
